cmd/common/image/inspect.go

```diff
@@ -157,6 +158,10 @@ func NewInspectCmd(runtime runtime.Runtime) *cobra.Command {
 		if jsonPath != "" && extraInfo == "" {
 			return fmt.Errorf("jsonpath %q can only be used with extra info", jsonPath)
 		}
+		showDataSources, _ := cmd.PersistentFlags().GetBool("show-datasources")
+		if showDataSources {
+			customResult = info.DataSources
```

Comment:
`--show-datasources` and `--extra-info` should be mutually exclusive. Otherwise this command is confusing:
```
$ sudo ig image inspect advise_seccomp:latest --show-datasources --extra-info=oci.metadata
```